DFE-Digital/early-years-foundation-reform
# HFEYP 36 CMS - Change Log out to take user to sign in page

## Ticket and context

Ticket: [HFEYP-36](https://dfedigital.atlassian.net/browse/HFEYP-36)
## Tech review

Added code so user is redirected to sign in screen when they sign out, and the next screen is the page they were previously on

### Code quality checks

## Product review

### How can someone see it in review app?
1. Click the [link to review app ](https://eyfs-cms-review-pr-572.london.cloudapps.digital)(posted by a `github-actions` bot below)
2. Sign in
3. Sign out
4. Verify that the page reached on sign out is the sign in page
